server/src/main/java/org/opensearch/cluster/ClusterServiceRequest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cluster;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.transport.TransportRequest;
+
+import java.io.IOException;
+
```

Comment:
As part of #221 and #2868 we need everyone to start added public and protected level class, enum, and interface javadocs. This is currently a herculean effort and I need folks to start picking up some of the slack on new commits.

Reply:
Great callout. I have added javadocs for the new files created in this PR. ]